src/components/Column.js

```diff
@@ -0,0 +1,94 @@
+const React = require('react');
+
+const Column = React.createClass({
+  propTypes: {
+    breakLg: React.PropTypes.number,
+    breakMd: React.PropTypes.number,
+    breakSm: React.PropTypes.number,
+    colLg: React.PropTypes.number,
+    colMd: React.PropTypes.number,
+    colSm: React.PropTypes.number,
+    columnCount: React.PropTypes.number,
+    offLg: React.PropTypes.number,
+    offMd: React.PropTypes.number,
+    offSm: React.PropTypes.number,
+    ordLg: React.PropTypes.number,
+    ordMd: React.PropTypes.number,
+    ordSm: React.PropTypes.number,
```

Comment:
Not a good one, no. My first inclination was to have full prop names ie:`columnSmall`. Wondered if that would be a lot for the parent component, but I think I agree about more discoverable over less typing.

I will change that are re-push.